db/migration/1722415645057-AddChartConfigHash.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddChartConfigHash1722415645057 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`
+            ALTER TABLE chart_configs
+            ADD COLUMN fullMd5 CHAR(24);
```

Comment:
I had thought about it. We agreed now that the grapher admin api should be the only place that writes grapher configs so we can make sure that everything is as we want it (and mojmir e.g. changed the chart diff tool to use this api) and so I decided against doing this in the db as a generated column. 

But the default is an interesting safety fallback. Let's go for that.

Reply:
I wrote this on Slack already, but putting it here too:

> MySQL also discards extra whitespace between keys, values, or elements in the original JSON document, and leaves (or inserts, when necessary) a single space following each comma (,) or colon (:) when displaying it.
> To make lookups more efficient, MySQL also sorts the keys of a JSON object.
> --https://dev.mysql.com/doc/refman/8.0/en/json.html#json-normalization

tl;dr MySQL will reorder JSON properties and change whitespace, making it so that a MySQL-generated hash and a JS-generated hash will likely be different. 🦩

Reply:
Another note on this: It would be nice if `fullMd5` would be `NOT NULL`, if that's at all feasible.

One way to do this is to first `ADD COLUMN`, then `UPDATE SET fullMd5 = ...`, and then `ALTER TABLE ALTER COLUMN ... NOT NULL`.